sigle/src/modules/settings/SettingsForm.tsx

```diff
@@ -88,6 +90,8 @@ export const SettingsForm = ({ settings, username }: SettingsFormProps) => {
       siteDescription: settings.siteDescription || '',
       siteColor: settings.siteColor || '',
       siteLogo: settings.siteLogo || '',
+      siteUrl: settings.siteUrl || '',
```

Comment:
Can you add some validation for siteUrl please? Right now I can enter `leo` and it will work, let's check that the formatting contains a valid TLD :)